package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "featuredfilm",
+  "version": "1.0.0",
+  "description": "Module to display featured film on Moodu",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "webpack-watch": "webpack -d --watch",
+    "start": "nodemon server/index.js",
+    "seed": "node server/seeder.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fec1-arwen/FeaturedFilm.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fec1-arwen/FeaturedFilm/issues"
+  },
+  "homepage": "https://github.com/fec1-arwen/FeaturedFilm#readme",
+  "devDependencies": {
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
```

Comment:
Stop using configs you've been provided and use the official documentation to set up webpack and babel.